addon/helpers/custom-taxonomy-filter.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+var customTaxonomies = ['bepress'];
+
+export function customTaxonomyFilter(params/*, hash*/) {
+    let words = params[0].split('/');
+    if (customTaxonomies.indexOf(words[0]) !== -1 && words.length > 1) {
```

Comment:
I think, instead of looking for specific taxonomies, we'll just have to assume, if it has a slash in it, it's a custom taxonomy.